app/vmctl/backoff/backoff.go

```diff
@@ -30,12 +24,26 @@ type Backoff struct {
 }
 
 // New initialize backoff object
-func New() *Backoff {
+func New(retries int, factor float64, minDuration time.Duration) *Backoff {
 	return &Backoff{
-		retries:     backoffRetries,
-		factor:      backoffFactor,
-		minDuration: backoffMinDuration,
+		retries:     retries,
+		factor:      factor,
+		minDuration: minDuration,
+	}
+}
+
+// Validate checks backoff object for correct values
+func (b *Backoff) Validate() error {
```

Comment:
Why not validating params in New() function instead?

Reply:
will check, and if needed update the logic

Reply:
made changes, added tests